pkg/kubewarden/chart/kubewarden/policy-server/SecurityContexts.vue

```diff
@@ -0,0 +1,75 @@
+<script>
+import { _CREATE } from '@shell/config/query-params';
+
+import Loading from '@shell/components/Loading';
+import ContainerConfiguration from './SecurityContexts/ContainerConfiguration.vue';
+import PodConfiguration from './SecurityContexts/PodConfiguration.vue';
+
+export default {
+  props: {
+    mode: {
+      type:    String,
+      default: _CREATE
+    },
+
+    value: {
+      type:     Object,
+      default:  () => {},
+      required: true
+    }
+  },
+
+  components: {
+    Loading,
+    ContainerConfiguration,
+    PodConfiguration,
+  },
+
+  async fetch() {},
+
+  data() {
+    return {
+      containerConfiguration: this.value.securityContexts.container,
+      podConfiguration:       this.value.securityContexts.pod
+    };
+  },
+
+  methods: {
+    updateContainerConfig(val) {
+      this.$emit('update-security-contexts', { type: 'container', data: val });
+    },
+    updatePodConfig(val) {
+      this.$emit('update-security-contexts', { type: 'pod', data: val });
+    },
+  },
+
+  computed: {
+    disabledByOsWindows() {
+      return !this.value.os?.name === 'windows';
+    },
+    disabledByOsLinux() {
+      return !this.value.os?.name === 'linux';
+    },
```

Comment:
Is the `os` property supplied by the policy server `spec`?

Reply:
Yep, as it's stated in multiple places in the docs, although I haven't seen it populated

<img width="1053" alt="Screenshot 2024-02-26 at 09 53 18" src="https://github.com/rancher/kubewarden-ui/assets/97888974/6aa89890-0842-44cb-bb22-44c38c3cc951">